lib/stemcell/metadata_source.rb

```diff
@@ -43,7 +43,7 @@ def default_options
       DEFAULT_OPTIONS.merge(config.default_options)
     end
 
-    def expand_role(role, environment, override_options={}, options={})
+    def expand_role(role, environment, override_keys=[], override_options={}, options={})
```

Comment:
If you add this as the last optional param that would make it backwards compatible?